src/Mvc/test/WebSites/BasicWebSite/RazorComponents/FetchData.razor

```diff
@@ -0,0 +1,53 @@
+﻿@using BasicWebSite.Services
```

Comment:
Wow, this is the first real `.razor` file in the whole world! 🥇

Reply:
heh. We already have tests for this in the razor repo. This might be the first one to actually **run** code.